src-ui/app/components/device-list/device-list-item/device-list-item.component.ts

```diff
@@ -62,6 +62,13 @@ export class DeviceListItemComponent implements OnInit {
     return 'unknown';
   }
 
+  protected get deviceRole(): string | null {
+    if (this.ovrDevice && this.ovrDevice.handleType) {
```

Comment:
If `handleType` is not optional, it should not have to be checked for truthiness, so either this check has to go, or `handleType` should become a  optional property.